.github/workflows/scripts/local-env.sh

```diff
@@ -13,4 +16,12 @@ export main_branch=master
 
 export DRY_RUN=true
 
-source "$(git rev-parse --show-toplevel)/.github/workflows/scripts/common.sh"
+SCRIPTS_ROOT="$(git rev-parse --show-toplevel)/.github/workflows/scripts"
+
+# Supress shellcheck false warning:
+# shellcheck source=/dev/null
+source "$SCRIPTS_ROOT/common.sh"
+
+SCRIPT="$SCRIPTS_ROOT/$1.sh"
```

Comment:
This is quite brave :) Shall we first check if user provided `$1` (can be added later if you like).